README.md

```diff
@@ -638,6 +653,8 @@ Excluding the content of the files, a SAML auth-config map data section may look
 apiVersion: v1
 data:
   auth-type: saml
+  auth-configuration: saml
```

Comment:
What is the distinction between `auth-type` and `auth-configuration` in what situations will they not be the same value?

Maybe it would be worth making the example one of those cases to avoid confusion?

Reply:
👍 I've added an auth-type / auth-configuration specification matrix table.

Reply:
Thanks, the table is really helpful. It allows me to ask better questions :wink: haha